platform/kube/inject/testdata/daemonset.yaml.injected

```diff
@@ -0,0 +1,87 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  annotations:
+    alpha.istio.io/sidecar: injected(deprecated)
+    status.sidecar.istio.io: injected-version-12345678
+  creationTimestamp: null
+  name: hello
+spec:
+  template:
+    metadata:
+      annotations:
+        alpha.istio.io/sidecar: injected(deprecated)
+        status.sidecar.istio.io: injected-version-12345678
+      creationTimestamp: null
+      labels:
+        app: hello
+        tier: backend
+        track: stable
+    spec:
+      containers:
+      - image: fake.docker.io/google-samples/hello-go-gke:1.0
+        name: hello
+        ports:
+        - containerPort: 80
+          name: http
+        resources: {}
+      - args:
```

Comment:
Feature request: allow user to provide a template config - they may want other -v, cpu/mem requirements, etc.
(same for initializer - but probably just the args need to be customized). 

If resource constraints are enabled, it's tricky to schedule jobs if you can't control the injected containers.